x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch.tsx

```diff
@@ -294,24 +295,53 @@ export const ChartSwitch = memo(function ChartSwitch(props: Props) {
                     prepend: (
                       <EuiIcon className="lnsChartSwitch__chartIcon" type={v.icon || 'empty'} />
                     ),
-                    append:
-                      v.selection.dataLoss !== 'nothing' ? (
-                        <EuiIconTip
-                          aria-label={i18n.translate('xpack.lens.chartSwitch.dataLossLabel', {
-                            defaultMessage: 'Warning',
-                          })}
-                          type="alert"
-                          color="warning"
-                          content={i18n.translate('xpack.lens.chartSwitch.dataLossDescription', {
-                            defaultMessage:
-                              'Selecting this chart type will result in a partial loss of currently applied configuration selections.',
-                          })}
-                          iconProps={{
-                            className: 'lnsChartSwitch__chartIcon',
-                            'data-test-subj': `lnsChartSwitchPopoverAlert_${v.id}`,
-                          }}
-                        />
-                      ) : null,
+                    append: (
+                      <EuiFlexGroup gutterSize="xs">
+                        {v.showBetaBadge ? (
+                          <EuiFlexItem grow={false}>
+                            <EuiBetaBadge
+                              label={
+                                <FormattedMessage
+                                  id="xpack.lens.chartSwitch.betaLabel"
+                                  defaultMessage="Beta"
+                                />
+                              }
+                              title={i18n.translate('xpack.lens.chartSwitch.betaLabel', {
+                                defaultMessage: 'Beta',
+                              })}
+                              tooltipContent={
+                                <FormattedMessage
+                                  id="xpack.lens.chartSwitch.betaTooltipContent"
+                                  defaultMessage="This chart type is not GA."
+                                />
+                              }
+                            />
```

Comment:
The `EuiBetaBadge` and its accompanying tooltip feel a bit out of place within an `EuiSelectable`. Any chance we could change this to a simple `EuiBadge` component?

Reply:
Replaced `EuiBetaBadge` with `EuiBadge` in https://github.com/elastic/kibana/pull/97978/commits/f90fe8f5204e1a520353b5b8db1cd2c2a35661c3